tests/tt_metal/multihost/single_host_mp_tests/test_context.cpp

```diff
@@ -258,12 +258,12 @@ TEST(DistributedContextExtraTest, IsendIrecvWait) {
     if (*ctx->rank() == 0) {
         auto req = ctx->isend(
             tt::stl::as_writable_bytes(tt::stl::Span<int>{buf_send.data(), buf_send.size()}), Rank{1}, Tag{7});
-        auto status = req->wait();
+        [[maybe_unused]] auto status = req->wait();
```

Comment:
... looks like always unused?  Why not:

```suggestion
        req->wait();
```